File: CPP/Disk.cpp
#include <iostream>
#include <string>

using namespace std;

class Disk
{
    private:
        string type;
        int capacity;
        int price;

    public:
        
        //constructors
        Disk(){}
        Disk(string x, int y, int z)
        {
            setType(x);
            setCapacity(y);
            setPrice(z);
        }
        
        //Type's methods
        void setType(string x)
        {
            this->type = x;
        }
        string getType()
        {
            return this->type;
        }

        //Capacity's methods
        void setCapacity(int y)
        {
            this->capacity = y;
        }
        int getCapacity()
        {
            return this->capacity;
        }

        //Price's methods
        void setPrice(int z)
        {
            this->price = z;
        }
        int getPrice()
        {
            return this->price;
        }

        //destructor
        ~Disk(){}
};

File: Java/Disk.java
public class Disk
{
    private String type;
    private int capacity;
    private int price;

    //constructor
    public Disk(){}
    public Disk(String x, int y, int z)
    {
        setType(x);
        setCapacity(y);
        setPrice(z);
    }

    //type's methods
    public void setType(String x)
    {
        this.type = x;
    }
    public String getType()
    {
        return this.type;
    }

    //capacity's methods
    public void setCapacity(int y)
    {
        this.capacity = y;
    }
    public int getCapacity()
    {
        return this.capacity;
    }

    //price's methods
    public void setPrice(int z)
    {
        this.price = z;
    }
    public int getPrice()
    {
        return this.price;
    }

}